ui/.env.development

```diff
@@ -3,3 +3,5 @@ REACT_APP_VERSION=$npm_package_version
 REACT_APP_NAME=$npm_package_name
 REACT_APP_SENTRY_DSN=https://10930d3b0a8d427cad2147d6d845be56@o226447.ingest.sentry.io/5721090
 REACT_APP_SENTRY_ENABLED=false
+# RELEASE_VERSION is set in config-overrides.js
+RELEASE_VERSION=unknown
```

Comment:
Prefer to be REACT_APP_XXX since environment variables not starting with `REACT_APP` are considered to be ignored in CRA.